tap_stripe/__init__.py

```diff
@@ -371,10 +382,13 @@ def reduce_foreign_keys(rec, stream_name):
     return rec
 
 
-def paginate(sdk_obj, filter_key, start_date, end_date, limit=100):
+def paginate(sdk_obj, filter_key, start_date, end_date, stream_name, limit=100):
```

Comment:
Explain why you have to include the stream_name as a new parameter?

Reply:
The `expand` parameter is passed in SDK request in the function and the required values are in the STREAM_TO_EXPAND_FIELDS map. So `stream_name` is used to retrieve expand's a value from that map.
https://github.com/singer-io/tap-stripe/blob/1223197217ab7c8a42c96d39474750bccf838dfd/tap_stripe/__init__.py#L389-L391